packages/kit/postinstall.js

```diff
@@ -18,7 +17,11 @@ try {
 			const packages = Array.isArray(pkg.workspaces) ? pkg.workspaces : pkg.workspaces.packages;
```

Comment:
```suggestion
			// it can be a glob pattern https://classic.yarnpkg.com/lang/en/docs/workspaces/#toc-tips-tricks
			const patterns = Array.isArray(pkg.workspaces) ? pkg.workspaces : pkg.workspaces.packages;
```

Reply:
What do you think about instead changing line 16 & 17 to
```js
			// Find all NPM and Yarn workspace glob patterns
			// https://classic.yarnpkg.com/blog/2018/02/15/nohoist/
			// https://docs.npmjs.com/cli/v9/configuring-npm/package-json#workspaces
			const patterns = Array.isArray(pkg.workspaces) ? pkg.workspaces : pkg.workspaces.packages;
```

Reply:
sure

Reply:
I changed that comment, and also renamed "directories" -> "workspaces" as I think that makes more sense here too